mermithid/processors/misc/BinnedDataFitter.py

```diff
@@ -0,0 +1,169 @@
+'''
```

Comment:
This looks more like a fitter than a misc...
Consider creating a new folder that would contain other fitters or placing this processor in the sampling folder (this last option might be confusing, but the RooFit fitter is already there, so .......)

Reply:
moved to Fitters